core/src/main/java/io/apicurio/hub/api/codegen/OpenApi2Quarkus.java

```diff
@@ -43,6 +43,11 @@ public OpenApi2Quarkus() {
         super();
     }
 
+    @Override
+    protected String getResourceName(String name) {
+        return "_OpenApi2Quarkus/" + name; // simpleName fails in the tests with Java 17
```

Comment:
this was constantly failing on my machine 🤷

Reply:
Apparently, this is not completely fixing the issue:
https://github.com/Apicurio/apicurio-codegen/actions/runs/6563277742/job/17827118036

Reply:
The problem is that the codegen produces different output with jdk 17 vs 11.  So it's failing because the expected output is different from the actual.  The pom.xml includes the target java version.

Reply:
fixed, we should revisit the "file by file"/"line by line" approach for those tests, but it should be ok for this iteration.